### DISCUSSION TITLE
Android spells hotkeys using touch controls

### DISCUSSION THREAD

**Discussion by SkoDave** [diasurgical/DevilutionX#5397] [Q&A]
Is there a way to quickly change spells only with touch controls? Will be added in the future, maybe like the 4 standard controls?

**Comment by AJenbo** [comment]
fastest current way is hitting the spell icon and then the spell. Or use a gamepad/keyboard.

the interface will eventually be improved

**Comment by UberBreaker** [comment]
Hello,

Thanks for the nice job doing Devilutionx.

Is there any way to have hotkeys for lauching different spells with the virtual gamepad on Android ?

**Reply by AJenbo** [reply]
no not yet
